examples/hooks/interfaces

```diff
@@ -46,8 +49,28 @@ auto lo
 iface lo inet loopback
 
 auto eth0
-iface eth0 inet dhcp
+EOF
 
+  if [ -n "$NIC_0_IP" ]; then
+    if [ -z "$NIC_0_NETWORK_SUBNET" ] ; then
+      echo "Missing NETWORK SUBNET from gnt-network"
+      exit 1
+    fi
+    if [ -z "$NIC_0_NETWORK_GATEWAY" ]; then
+      echo "Missing NETWORK GATEWAY from gnt-network"
+      exit 1
+    fi
+    cat >> $TARGET/etc/network/interfaces <<EOF
+iface eth0 inet static
+  address $NIC_0_IP
+  netmask ${NIC_0_NETWORK_SUBNET##*/}
```

Comment:
Question: I don't know what value NIC_0_NETWORK_SUBNET will have. Is is something like 192.168.0.0/16? If so, does it mean that here will we have "netmask 16"?

If so, I didn't know this works. The `ifup` command behaves differently between 255.255.255.0 and 24, for some strange reason: in the first case, passes explicit broadcast address to `ip`, in the second, it passes `+`, which has special significance (auto-compute). I wonder why it is different… anyway, it works.